### ISSUE TITLE
ActionView::Template::Error (undefined method `administrator' for nil:NilClass)

### ISSUE THREAD

**Issue by eitoball** [Safecast/safecastapi#555]
https://safecast.slack.com/archives/C026F2Q1U/p1558969525010600

```
[7427db88-6ddc-4e66-bad6-476b609384a0] [52.68.50.238]
ActionView::Template::Error (undefined method `administrator' for nil:NilClass):
    72:     <%= render 'send_email' -%>
    73:   </div>
    74:   <%- if @bgeigie_import.status == 'awaiting_response' -%>
    75:     <div class="tab-pane" id="contact-detail">
    76:       <%- history = @bgeigie_import.uploader_contact_histories.last %>
    77:       Contacted by: <%= history.administrator.name %><br>
    78:       Contacted on: <%= history.created_at %><br>
  app/views/bgeigie_imports/show.html.erb:75:in `_app_views_bgeigie_imports_show_html_erb__738638169518485332_47139259850160'
  app/controllers/bgeigie_imports_controller.rb:101:in `show'
```

**Comment by matschaffer** [comment]
Seems like what's happening here is we have some imports in `awaiting_response` status that don't have any `uploader_contact_histories` associations.

Also notable that `uploader_contact_histories` table is empty so could indicate some bug in getting those stored.

**Comment by eitoball** [comment]
Fixed in #558
